CMakeLists.txt

```
@@ -66,61 +67,62 @@ if(DEFINED QT_VERSION)
   set(CMAKE_AUTOUIC ON)
 
   set(
-      VISUALIZER_CORE_SOURCES
-      ${CMAKE_SOURCE_DIR}/src/visualizer/settings/SettingsRegistry.cpp
+          VISUALIZER_CORE_SOURCES
+          ${CMAKE_SOURCE_DIR}/src/visualizer/settings/SettingsRegistry.cpp
   )
 
   set(
-      VISUALIZER_SOURCES
-      ${CMAKE_SOURCE_DIR}/src/visualizer/window/VisualizerWindow.cpp
-      ${CMAKE_SOURCE_DIR}/src/visualizer/vertex/Vertex.cpp
-      ${CMAKE_SOURCE_DIR}/src/visualizer/graphPanel/GraphPanel.cpp
+          VISUALIZER_SOURCES
+          ${CMAKE_SOURCE_DIR}/src/visualizer/window/VisualizerWindow.cpp
+          ${CMAKE_SOURCE_DIR}/src/visualizer/vertex/Vertex.cpp
+          ${CMAKE_SOURCE_DIR}/src/visualizer/vertexMenu/VertexMenu.cpp
+          ${CMAKE_SOURCE_DIR}/src/visualizer/graphPanel/GraphPanel.cpp
```

Same comment from l:39-l:41
```suggestion
     VISUALIZER_SOURCES
     ${CMAKE_SOURCE_DIR}/src/visualizer/window/VisualizerWindow.cpp
     ${CMAKE_SOURCE_DIR}/src/visualizer/vertex/Vertex.cpp
     ${CMAKE_SOURCE_DIR}/src/visualizer/vertexMenu/VertexMenu.cpp
     ${CMAKE_SOURCE_DIR}/src/visualizer/graphPanel/GraphPanel.cpp
```